packages/core/src/themes/buildTheme.ts

```diff
@@ -95,7 +95,7 @@ export default function buildTheme(
       },
       offscreen: {
         position: 'absolute',
-        left: -9999,
+        left: '-5vw',
```

Comment:
curious why `5`? instead of `50vw` or something a little larger?

Reply:
`5` is arbitrary to some extent, but mostly trying to minimize size. `offscreen` specifies a `width`/`height` of `1px` and `overflow: hidden`, so I'd guess `5vw` is okay. 

what do you think of `-10vw` to be safe but smaller than `-50vw`?

Reply:
good call! `5` sounds good then!

Reply:
@williaster Instead of using a negative left, I think it would be better to use a positive right > 100. That way the width of the element won't cause it to sometimes be within the viewport. You'll want to add a few vw to account for things that bleed, like box shadow. I'd go with:

```suggestion
        right: '105vw',
```